bin/release.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
-bundle exec rails db:migrate
-bundle exec rails feature_flags:initialize
+echo "Running release script" &&
+  bundle exec rails feature_flags:initialize
```

Comment:
should the db:migrate command be here as well?

Reply:
We are running the migrations in the entrypoint script. Rails does the same [here](https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/docker-entrypoint.tt#L12).